frontend/public/src/containers/ProductDetailEnrollApp.js

```diff
@@ -70,7 +71,7 @@ export class ProductDetailEnrollApp extends React.Component<
             <div className="flex-grow-1 align-self-end">
               Learn online and get a certificate
             </div>
-            <div className="text-right align-self-end">${product.price}</div>
+            <div className="text-right align-self-end">${product.price - flexPriceDiscountAmount}</div>
```

Comment:
The amount that's coming through here isn't always a dollar amount - they can be a set amount off, or they can be a percentage of the original amount or a fixed price. In the latter two cases, this calculation fails. For example, this was a $999 course that I ran through locally and set up with a 75% discount for the tier I was in: 
![Screen Shot 2022-07-21 at 12 35 41 PM](https://user-images.githubusercontent.com/945611/180277817-5db4c3cb-ab16-4517-a6fc-19e8acc61b61.png)

(This should be $249.75.)